puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -6,46 +6,28 @@
   String $password,
   Integer[0] $jenkins_jobs_update_timeout = 600,
   String $command_arguments = 'foreman-infra-jenkins-job-update',
+  String $git_project_name = 'jenkins-jobs',
+  String $git_repo = 'https://github.com/theforeman/jenkins-jobs.git',
+  Optional[String] $git_ref = undef,
 ) {
   $config_name = $name
   $directory = '/etc/jenkins_jobs'
   $inifile = "${directory}/jenkins_jobs_${config_name}.ini"
 
-  file { "${directory}/${config_name}":
-    ensure  => directory,
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0755',
-    recurse => true,
-    purge   => true,
-    force   => true,
-    source  => "puppet:///modules/jenkins_job_builder/${config_name}",
```

Comment:
You can remove these files now. Probably best in a separate commit but can be in the same PR.